packages/@react-spectrum/s2/src/ProgressCircle.tsx

```diff
@@ -54,17 +56,52 @@ const track = style({
     default: 'gray-300',
     isStaticColor: 'transparent-overlay-300',
     forcedColors: 'Background'
+  },
+  strokeWidth: {
+    default: `[${pxToRem(3)}]`,
+    size: {
+      S: 2,
+      L: 4
```

Comment:
I don't think these other values are converted to rems in the theme. Borders and strokes use pixels by default, but we chose to use rems for progress circle. So I think you will need to manually convert these too.